package.json

```diff
@@ -9,15 +9,23 @@
     "async": "~0.9.0",
     "aws-sdk": "^2.0.25",
     "babel": "^5.5.4",
+    "bl": "~1.0.0",
```

Comment:
Modules that were missing when I was setting up my local copy. I installed them one by one but thought others may find it helpful if the package file gets updated, so they get installed automatically.
Nothing to do with this particular issue though, so up to you whether to ignore them :)